openedx/features/journals/urls.py

```diff
@@ -2,10 +2,12 @@
 Defines URLs for course bookmarks.
 """
 
+from django.conf import settings
 from django.conf.urls import url
 
 from openedx.features.journals.views.marketing import bundle_about
 from openedx.features.journals.views import learner_dashboard
+from openedx.features.journals.views.journal_page import render_xblock_by_journal_access
```

Comment:
lets change the name of the view from `openedx.features.journals.views.journal_page` to `openedx.features.journals.views.journal_xblock`